src/Bicep.LangServer.IntegrationTests/DefinitionTests.cs

```diff
@@ -233,6 +233,54 @@ public async Task GoToDefinition_onLoadFunctionArgument_shouldNotThrow(string fu
             files);
         }
 
+        [TestMethod]
+        public async Task Goto_definition_works_with_using_statement()
+        {
+            using var server = await MultiFileLanguageServerHelper.StartLanguageServer(TestContext, services => services.WithFeatureOverrides(new(ParamsFilesEnabled: true)));
+            var helper = new ServerRequestHelper(TestContext, server);
+
+            var (contents, cursor) = ParserHelper.GetFileWithSingleCursor(@"
+using 'main.bi|cep'
+");
+            var (bicepContents, bicepCursors) = ParserHelper.GetFileWithCursors(@"||
```

Comment:
Does `||` represent part that would be highlighted?

Reply:
Each `|` will give me a different cursor - so this'll give me cursor 0 at position 0 and cursor 1 at position 0.